Analysis/Tasks/PWGLF/CMakeLists.txt

```diff
@@ -84,6 +84,11 @@ o2_add_dpl_workflow(raacharged
                     PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore
                     COMPONENT_NAME Analysis)
 
+o2_add_dpl_workflow(raameasurement
```

Comment:
Let's not have measurement in the name. If this is identified RAA, maybe call it identified-raa